app/views/application/_feedback_link.html.haml

```diff
@@ -0,0 +1,2 @@
+.feedback{tabindex:1}
```

Comment:
Ah I missed it. That explains some other weird behaviour now. I can delete another tabindex later in the code too.